### PR TITLE
Drop fnv algorithm on Web first.

### PR DESCRIPTION
**Pull request by rebornix**
First step for https://github.com/microsoft/vscode-jupyter/issues/10358, dropping the fnv hash algorithm for the Web platform. Migration for the desktop version can come later.

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for feature-requests.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Comment by rchiodo** [comment]
Great way to solve this. I think this would be good enough. We can keep the fnv there for desktop.

File: src/platform/common/serviceRegistry.node.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export function registerTypes(serviceManager: IServiceManager) {
     serviceManager.addSingleton<ILanguageService>(ILanguageService, LanguageService);
     serviceManager.addSingleton<IBrowserService>(IBrowserService, BrowserService);
     serviceManager.addSingleton<IHttpClient>(IHttpClient, HttpClient);
-    serviceManager.addSingleton<ICryptoUtils>(ICryptoUtils, CryptoUtils);
+    serviceManager.addSingleton<ICryptoUtils>(ICryptoUtils, CryptoUtilsNode);
```
Comment (DonJayamanne) [inline src/platform/common/serviceRegistry.node.ts:76]: Looking at the PR title, i don't see any code where we're dropping the use of the old fnv liv for hashing on the web.
This PR merely introduces a dedicated method for desktop and not the former.

**Review by DonJayamanne — APPROVED** [review]
I'd suggest changing the title of this PR before merging this - to reflect the fact that its actually not removing the fnv algo for web, but introducing a dedicated class for desktop

File: src/platform/common/serviceRegistry.node.ts (hunk 1 above)
Comment (rebornix) [reply src/platform/common/serviceRegistry.node.ts:76]: `CryptoUtils` is now only used in the Web and it doesn't use `fnv` anymore.

**Pull request merged**